src/network/org.freedesktop.network1.conf

```diff
@@ -20,22 +20,7 @@
         </policy>
 
         <policy context="default">
-                <deny send_destination="org.freedesktop.network1"/>
-
-                <allow send_destination="org.freedesktop.network1"
-                       send_interface="org.freedesktop.DBus.Introspectable"/>
-
-                <allow send_destination="org.freedesktop.network1"
-                       send_interface="org.freedesktop.DBus.Peer"/>
-
-                <allow send_destination="org.freedesktop.network1"
-                       send_interface="org.freedesktop.DBus.Properties"
-                       send_member="Get"/>
-
-                <allow send_destination="org.freedesktop.network1"
-                       send_interface="org.freedesktop.DBus.Properties"
-                       send_member="GetAll"/>
-
+                <allow send_destination="org.freedesktop.network1"/>
```

Comment:
hmm, this opens things for everybody, widely, is that really intended?

Reply:
Sorry, this is quite early and WIP version, so I was lazy to list up methods.